src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -628,7 +627,7 @@ private void InitializeInterruptDriver()
     {
         try
         {
-            _interruptDriver = new LibGpiodV1Driver(0);
+            _interruptDriver = new LibGpiodDriver(0);

```

Comment:
should this be `()`?

Reply:
I think it's more clear to keep it explicit here. This is for RPI3 and 4 only.